pipelines/pom.xml

```diff
@@ -160,6 +161,30 @@
         <version>${logback.version}</version>
         <scope>runtime</scope>
       </dependency>
+
+      <dependency>
+        <groupId>org.apache.parquet</groupId>
+        <artifactId>parquet-avro</artifactId>
+        <version>${parquet.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.apache.parquet</groupId>
```

Comment:
These dependencies are transitive dependencies of `beam-sdks-java-io-parquet` and were resolving to a different version than the `parquet-avro`. So, locking all the dependencies to the same version at the root package.

Reply:
I think this is okay but one problem that I see is that Beam's code may start throwing exceptions if we use a difference Parquet lib version. Although this issue is not unique here, it might be worth trying to fix avoid it. One option is to remove these new deps and only change `parquet.version` when we change Beam's version and keep them consistent; WDTY?

Reply:
Agreed, I have rolled back the changes and just added a comment to keep the versions consistent across module. Thanks!

Reply:
Great, thanks, then please update the PR title as well.

Reply:
Done